src/new-apps/components/AppListCard/AppListCard.tsx

```diff
@@ -0,0 +1,132 @@
+import { Card, CardActions, CardContent, Typography } from "@material-ui/core";
```

Comment:
This component is pretty big, can you break it down? 

also please write unit test for this file